hca/util/__init__.py

```diff
@@ -382,6 +382,7 @@ def logout(self):
         Clear {prog} authentication credentials previously configured with ``{prog} login``.
         """
         try:
+            del self.config["application_secrets"]
```

Comment:
If this line results in a KeyError, will `del self.config["oauth2_token"]` still run?

Reply:
good catch, updated